libraries/rush-lib/src/schemas/pnpm-config.schema.json

```diff
@@ -165,6 +165,11 @@
       "description": "This option overrides the resolution-mode in PNPM. Use it if you want to change the default resolution behavior when installing dependencies. Defaults to \"highest\".\n\nPNPM documentation: https://pnpm.io/npmrc#resolution-mode.",
       "type": "string",
       "enum": ["highest", "time-based", "lowest-direct"]
+    },
+
+    "autoInstallPeers": {
+      "description": "This option overrides the auto-install-peers in PNPM. Use it if you want to change the default resolution behavior when installing dependencies. Defaults to \"highest\".\n\nPNPM documentation: https://pnpm.io/npmrc#auto-install-peers.",
+      "type": "boolean"
```

Comment:
We also need to update the `rush init` template:

[rush-lib/assets/rush-init/common/config/rush/pnpm-config.json](https://github.com/microsoft/rushstack/blob/40c63c7e41b2466ab46f9d30ce82c8bfd37f9543/libraries/rush-lib/assets/rush-init/common/config/rush/pnpm-config.json#L44)

I can help with the docs